akka-sample-javaslang/src/main/java/sample/javaslang/Master.java

```diff
@@ -0,0 +1,54 @@
+package sample.javaslang;
+
+import akka.actor.Props;
+import akka.actor.UntypedAbstractActor;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+
+import static sample.javaslang.Minion_StoppedPatterns.*;
+
+import static javaslang.API.*;
+import static javaslang.Predicates.*;
+
+public class Master extends UntypedAbstractActor {
+
+  private final Maze maze;
+
+  private List<Minion.Stopped> results = List.empty();
+
+  public Master(Maze maze) {
+    this.maze = maze;
+  }
+
+  public void onReceive(Object message) {
```

Comment:
It would be possible but would also require a similar convertor to be implemented in the sample. I think it would be a bit of a cognitive distractions of how `Match` from Javaslang can be used. That is why I have sculpted the sample to have three different uses of `Match` each being a bit more advanced and using more of the power than the other one.